docs/reference/search/search.asciidoc

```diff
@@ -480,6 +480,39 @@ A boost value greater than `1.0` increases the score. A boost value between
 `0` and `1.0` decreases the score.
 ====
 
+experimental::[]
```

Comment:
Is  the new`knn` section within `_search` API is also experimental?
I was under impression that it is not experimental anymore.

 cc @giladgal

Reply:
In our discussions, we said that integrating ANN into the `_search` endpoint is a separate step from making the feature GA. Before GA, we still have a few items to tackle (like adding telemetry, improving index performance, etc.)